src/settings.h

```diff
@@ -37,6 +38,8 @@ typedef struct
 typedef struct
 {
     uint16_t port;
+    char *database_file;
+    rest_list_t *security;
```

Comment:
I would suggest moving device list to core (rest_t), refactor `database_init(coap_settings_t *coap)` to `database_load_file(const char *file)` and call it from rest_init.

Reply:
I can do that, but database_load_file will still need a pointer to the whole rest_context_t structure because it needs to have the ability to change the address of the device list

Reply:
Also database_file is used by /devices callbacks, so it will also be included in rest_context_t